mmcls/models/backbones/resnet.py

```diff
@@ -247,13 +294,22 @@ def _inner_forward(x):
             out = self.norm1(out)
             out = self.relu(out)
 
+            if self.with_plugins:
+                out = self.forward_plugin(out, 'after_conv1')
+
             out = self.conv2(out)
             out = self.norm2(out)
             out = self.relu(out)
 
+            if self.with_plugins:
```

Comment:
How about we hide the judgment of self.with_plugins inside self.forwawrd_plugin?